### DISCUSSION TITLE
Dudas sobre segunda entrega TPA

### DISCUSSION THREAD

**Discussion by mercedesmagnelli** [dds-jv/foro#236] [TPA - 2021] [answered]
Buen día! 
Nos encontramos con un par de dudas al comenzar a resolver la segunda entrega del TP anual que nos trabaron para poder continuar: 

1. Es correcto llegar al mismo dato de dos formas distintas? Porque nosotros modelamos que las asociaciones conozcan a las mascotas registradas en ellos e indirectamente podemos conocer los dueños (porque las mascotas conocen a su dueño, no el dueño a la mascota) y después en la asociación tenemos una lista de dueños. Tendríamos que optar por una única forma de acceder a esta información? 

2. El voluntario debe estar registrado en el sistema? O sea, tiene que tener usuario y contraseña? Además, las asociaciones tienen que conocer a sus voluntarios o el voluntario conocer a la asociación a la que pertenece?

3. Es necesario tener nuevas clases "Mascota con chapita"  y "Mascota sin chapita" o es preferible reutilizar de alguna forma las clases que ya tenemos?

¡Muchas gracias!

**Comment by nanderson87** [comment, marked as answer]
Hola @mercedesmagnelli:

Por lo que puedo ver la mayoría de estás dudas estan fuertemente relacionadas a su TPA y con decisiones que fueron tomando en la entrega anterior. 
Por ejemplo, para la pregunta 3, puede darse el caso que otro equipo por ahi ni modelo el concepto de mascota y es por este motivo que los invito a canalizar estas mismas dudas directamente con su ayudante (o ayudantes). Ya  que conoce más su proyecto y van a poder debatir mejor estas decisiones y ver cuales son los beneficios de ir por un camino u el otro o si algo tiene más sentido o menos con lo que ya fueron modelando.

La única duda que me parece más general es la del punto 2. Y a mi parecer tiene sentido que el voluntario sea un usuario más que simplemente cumple un rol diferente. Pero nuevamente, no dejaría de discutir esto con el ayudante que tienen asignado.

**Reply by mercedesmagnelli** [reply]
Perfecto! Muchísimas gracias por la respuesta, nos vamos a estar comunicando con nuestro ayudante :)
